sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/RenderClient.java

```diff
@@ -0,0 +1,568 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.maps.render;
+
+import java.io.OutputStream;
+import java.util.Objects;
+
+import com.azure.core.annotation.ReturnType;
+import com.azure.core.annotation.ServiceClient;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.http.rest.Response;
+import com.azure.core.http.rest.SimpleResponse;
+import com.azure.core.http.rest.StreamResponse;
+import com.azure.core.models.GeoBoundingBox;
+import com.azure.core.util.BinaryData;
+import com.azure.core.util.Context;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.core.util.FluxUtil;
+import com.azure.maps.render.models.Copyright;
+import com.azure.maps.render.models.CopyrightCaption;
+import com.azure.maps.render.implementation.models.ErrorResponseException;
+import com.azure.maps.render.models.MapAttribution;
+import com.azure.maps.render.models.MapStaticImageOptions;
+import com.azure.maps.render.models.MapTileOptions;
+import com.azure.maps.render.models.MapTileset;
+import com.azure.maps.render.models.TileIndex;
+import com.azure.maps.render.models.TilesetId;
+
+/** Initializes a new instance of the synchronous RenderClient type. */
+/**
```

Comment:
There are 2 javadoc style comments here. There should only be one javadoc comment for the class.